api/src/send_email.py

```diff
@@ -0,0 +1,104 @@
+import datetime
+import logging
+import os
+from flask import abort, request
+from flask_restful import Resource
+from marshmallow import Schema
+from marshmallow import fields
+
+import smtplib, ssl
+
+class SendEmailSchema(Schema):
+    email = fields.Str(required=True)
+    subject = fields.Str(required=True)
+    message = fields.Str(required=True)
+
+class SendEmailResource(Resource):
+    options_headers = {
+        'Access-Control-Allow-Origin': '*',
+        'Access-Control-Allow-Headers': 'Content-Type, Authorization',
+        'Access-Control-Allow-Methods': 'POST', 'OPTIONS'
+        'Access-Control-Max-Age': '3600'
+    }
+
+    headers = {
+        'Access-Control-Allow-Origin': '*',
+        'Access-Control-Allow-Headers': 'Content-Type, Authorization',
+        'Access-Control-Allow-Methods': 'POST', 'OPTIONS'

```

Comment:
Does python know to combine 'POST' and 'OPTIONS' as a list? Normally you make the 'Access-Control-Allow-Methods' have a value of 'POST, OPTIONS' as a single string with commas. We should probably switch it to a single string with commas even if it does work since that would be consistent with our existing endpoints.

Reply:
These have been made into a single string.